applications/taleps/tasks.py

```diff
@@ -0,0 +1,80 @@
+from trquake.celery import app
+import requests, json
+
+from taleps.models import TalepAddress, TalepLocation
+
+# @app.task
+def collect_taleps():
```

Comment:
Bu task simdilik aktif degil, amaci ayni tweet toplamak gibi bulunan farkli sitelerden cronla veri toplayip postgres'e ingest etmesi. Simdilik proof of concept olarak https://www.depremyardim.com ustunde calistim, trendyol API ve celery hakkinda pek bilgim olmadigi icin.